index.php

```diff
@@ -166,8 +166,13 @@ function gtag() {
         <div data-image="<?php siteUrl(); ?>images/backgrounds/mountains.jpg" class="slide bg-mask background-image full-vh">
             <div class="container-slide vertical-align center head-desc">
                 <div class="container">
-                    <div class="row">
-                        <div class="col-md-12">
+                    	<div class="row">
+				<div class="col-md-12">
+				<div style="padding: 15px 0px 20px 0px;"><a href="https://telegra.ph/BitcoinZ--contest-game-for-10-000-000-coins-BTCZ-12-20" target="_blank"><img style="display: inline; width: 70%;" src="<?php siteUrl(); ?>images/bitcoinz_contest.png"></a></div>
```

Comment:
Can you put these styles in the stylesheet?

Reply:
what for? this is a temporary banner.